test/show.jl

```diff
@@ -753,6 +753,9 @@ test_mt(show_f5, "show_f5(A::AbstractArray{T,N}, indices::Vararg{$Int,N})")
 @test_repr "[a;]"
 @test_repr "[a; b]"
 
+# Issue #32775, printing ; in a tuple
+@test sprint(show, :((a,;b))) == ":((a; b))"
```

Comment:
This printing is incorrect; `(a,;b)` and `(a; b)` are different (the comma makes it a tuple).

Reply:
Maybe I'm missing something, but wouldn't Julia parse `(a; b)` as a block?